style-guide/styles/wds-components/_toggles.scss

```diff
@@ -0,0 +1,76 @@
+@import '../wds-variables/colors';
+
+.wds-switch {
+	&__input {
+		display: none;
+	}
+
+	&__label {
+		cursor: pointer;
+		font-size: 16px;
+		padding: 7px 5px 7px 52px;
+		position: relative;
+
+		&:empty {
+			padding-left: 45px;
+		}
+
+		&:before, &:after {
+			content: '';
+			margin: 0;
+			outline: 0;
+			position: absolute;
+			top: 50%;
+			transform: translate(0, -50%);
+			transition: all 0.3s ease;
+		}
+
+		&:before {
+			background-color: $wds-color-light-c5;
+			border-radius: 8px;
+			height: 14px;
+			left: 6px;
+			width: 34px;
+		}
+
+		&:after {
+			background-color: $wds-color-light-c2;
+			border-radius: 50%;
+			box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.14), 0 2px 2px 0 rgba(0, 0, 0, 0.098), 0 1px 5px 0 rgba(0, 0, 0, 0.084);
```

Comment:
`.14` instead of `0.14`, `.098` instead of `0.098`, `.084` instead of `0.084`

Reply:
And if you want a shorter line you can use `rgba(#000, .14)` in SCSS